docs/releasing.md

```diff
@@ -24,6 +24,13 @@ Creating a federation v2 release involves the following steps:
    3. `kubefed2 version` (check that the output is as expected)
    4. `tar cvzf kubefed2.tar.gz kubefed2`
    5. `sha256sum kubefed2.tar.gz > kubefed2.tar.gz.sha`
+4. Package the helm chart for release
```

Comment:
Please update the instructions for kubefed to use `.tgz` instead of `.tar.gz` for parity with the chart archive name.

Reply:
Updated.